cookbooks/cdo-apps/templates/default/crontab.erb

```diff
@@ -124,6 +124,7 @@
       cronjob at:'0 2 * * *', do:deploy_dir('bin', 'cron', 'hoc_student_name_cleanup')
       cronjob at:'0 0 * * *', do:deploy_dir('bin', 'cron', 'send_permission_email_reminders')
       cronjob at:'*/5 * * * *', do:deploy_dir('bin', 'cron', 'monitor_projects')
+      cronjob at:'0	20 * * *', do:deploy_dir('bin', 'cron', 'cleanup_mvp_ai_tutor_interactions')
```

Comment:
Two notes:
- Can you switch this to spaces instead of tabs?
- This is 8pm UTC which is noon pacific. Is that what you intended? I don't think it really matters, but if you're aiming for non-peak time, this ain't it

Reply:
updated! Thanks for the notes. I don't think the timing is crucial, but 8pm pacific is what I intended 🤦‍♀️  and is much better traffic-wise.